src/scp/LocalNode.cpp

```diff
@@ -407,6 +407,52 @@ LocalNode::toJson(SCPQuorumSet const& qSet,
     return ret;
 }
 
+SCPQuorumSet
+LocalNode::fromJson(Json::Value const& qSetJson) const
+{
+    return fromJson(
+        qSetJson, [&](std::string const& s) { return mDriver.fromStrKey(s); });
+}
+
+SCPQuorumSet
+LocalNode::fromJson(Json::Value const& qSetJson,
+                    std::function<NodeID(std::string const&)> const toNodeID)
```

Comment:
Do we need toNodeID here? Seems like in all use cases we'd use the same fromStrKey function?

Reply:
I originally added this for symmetry with the `toJson` function that takes a similar function, but I agree that if we're not using this version it probably shouldn't exist. I removed it in 2b2ecb16c119848fd26b86ec60dd20601fe61e04.